Source/WebCore/style/UserAgentStyle.cpp

```diff
@@ -234,9 +234,7 @@ void UserAgentStyle::ensureDefaultStyleSheetsForElement(const Element& element)
         StringBuilder fullscreenRules;
         fullscreenRules.appendCharacters(fullscreenUserAgentStyleSheet, sizeof(fullscreenUserAgentStyleSheet));
         fullscreenRules.append(RenderTheme::singleton().extraFullScreenStyleSheet());
-        if (element.document().quirks().needsBlackFullscreenBackgroundQuirk())
-            fullscreenRules.append(":-webkit-full-screen { background-color: black; }"_s);
-        fullscreenStyleSheet = parseUASheet(fullscreenRules.toString());
```

Comment:
Sorry I wasn't clear, all these lines can be removed.

And these 3 lines too: https://searchfox.org/wubkat/rev/fde0f31b8e7849e3eb1bf6c82439b4b9a5d31baf/Source/WebCore/rendering/RenderTheme.h#96-98